model_signing/serialize_test.py

```diff
@@ -621,6 +627,12 @@ def test_different_ignored_paths(self):
         assert (r0 != r2)
         assert (r0 != r3)
         assert (r0 != r4)
+        assert (r1 != r2)
+        assert (r1 != r3)
+        assert (r1 != r4)
+        assert (r2 != r3)
+        assert (r2 != r4)
+        assert (r3 != r4)
```

Comment:
I'm think we could also put all these in a set and check the length of the set to be 5?